setup.py

```diff
@@ -25,33 +25,26 @@
     long_description = fh.read()
 
 dependencies = [
-    # Dependency corrections from our requirements
-    "attrs==23.1.0",
-    "grpcio==1.53.2",
-    "lazy-object-proxy==1.9.0",
-    "marshmallow==3.19.0",
-    # Core dependencies
-    "fsspec>=2022.8.2",
-    "google-api-python-client==2.91.0",
-    "ibis-framework==5.1.0",
-    "impyla==0.18.0",
-    "SQLAlchemy==1.4.49",
-    "PyMySQL==1.1.1",
-    "psycopg2-binary==2.9.6",
-    "PyYAML==6.0",
-    "pandas==1.5.3",
-    "proto-plus==1.22.3",
-    "pyarrow==14.0.1",
-    "pydata-google-auth==1.8.0",
-    "google-cloud-bigquery==3.11.3",
-    "google-cloud-bigquery-storage==2.20.0",
-    "google-cloud-spanner==3.36.0",
-    "google-cloud-storage==2.10.0",
-    "jellyfish==1.1.0",
-    "tabulate==0.9.0",
-    "Flask==2.3.2",
-    "parsy==2.1",
-    "google-cloud-secret-manager>=2.16.0",
+    "Flask>=2.2", # Some versions of airflow such as 2.9.1 depend on flask<2.3 and >=2.2
+    "fsspec>=2024.9.0",
+    "google-api-python-client>=2.144.0",
+    "google-cloud-bigquery>=3.25.0",
+    "google-cloud-bigquery-storage>=2.26.0",
+    "google-cloud-secret-manager>=2.20.2",
+    "google-cloud-spanner>=3.49.1",
+    "google-cloud-storage>=2.18.2",
+    "ibis-framework==5.1.0", # Pinned to 5.1.0, significant work to bump to 7.1.0
+    "impyla>=0.19.0",
+    "jellyfish>=1.1.0",
+    "pandas==2.0.3", # 2.03 is the highest version that still supports python 3.8
+    "parsy>=2.1",
+    "psycopg2-binary>=2.9.9",
+    "pyarrow==14.0.1", # ibis-framework 7.1.0 depends on pyarrow<15 and >=2
+    "pydata-google-auth>=1.8.2",
+    "PyMySQL>=1.1.1",
+    "PyYAML>=6.0.2",
+    "SQLAlchemy>=1.4.49",
```

Comment:
I tested this, changing to `SQLAlchemy==1.4.49` fixes the error when dealing with SQL syntax!

Reply:
Fixed here -> https://github.com/GoogleCloudPlatform/professional-services-data-validator/pull/1264/commits/44fd08ceaaf12ea10ae60858ab86fb883c9e3aa6